dotCMS/src/main/java/com/dotcms/rest/api/v1/experiment/ExperimentFactory.java

```diff
@@ -0,0 +1,233 @@
+            final List<Map<String, Object>> experimentsMap = new DotConnect()
+                    .setSQL(GET_RUNNING_EXPERIMENT_QUERY)
+                    .loadObjectResults();
+
+            return experimentsMap.stream()
+                    .map(experimentMap -> createExperiment(experimentMap))
+                    .collect(Collectors.toList());
+        } catch (DotDataException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @NotNull
+    private static Experiment createExperiment(Map<String, Object> experimentMap) {
+        final Experiment experiment = new Experiment(
+               experimentMap.get("key").toString(),
+               experimentMap.get("name").toString(),
+                ExperimentStatus.valueOf(experimentMap.get("status").toString()),
+               Boolean.parseBoolean(
+                       experimentMap.get("uniquePerVisitor").toString()),
+               Integer.parseInt(
+                       experimentMap.get("lookBackWindowMinutes").toString())
+       );
+
+        final List<Rule> rules = getRuleByExperiment(experiment);
+        final VariantsCollection variantsCollection = getVariantByExperiment(experiment,  experimentMap.get("pageId").toString());
+        final List<AnalyticEvent> events = getEventsByExperiment(experiment);
+
+        experiment.setEvents(events);
+        experiment.setRules(rules);
+        experiment.setVariantsCollection(variantsCollection);
+
+        return experiment;
+    }
+
+    private static List<AnalyticEvent> getEventsByExperiment(final Experiment experiment) {
+        try{
+            return new DotConnect()
+                    .setSQL(GET_EVENTS_BY_EXPERIMENT_QUERY)
+                    .addParam(experiment.getKey())
+                    .loadObjectResults()
+                    .stream()
+                    .map(map -> {
+                        try {
+                            return new AnalyticEvent(
+                                        AnalyticEventType.valueOf(map.get("event_key").toString()),
+                                        JsonUtil.toMap(map.get("parameters").toString())
+                                    );
+                        } catch (IOException e) {
+                            return null;
+                        }
+                    })
+                    .collect(Collectors.toList());
+        } catch (DotDataException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static List<Rule> getRuleByExperiment(final Experiment experiment) {
+        try {
+            return new DotConnect()
+                    .setSQL(GET_RULES_BY_EXPERIMENT_QUERY)
+                    .addParam(experiment.getKey())
+                    .loadObjectResults()
+                    .stream()
+                    .map(result -> result.get("rule_id").toString())
+                    .map(ruleID -> {
+                        try {
+                            return APILocator.getRulesAPI()
```

Comment:
An API being called here, probably this would be responsibility of the ExperimentAPI, Factory do not see API, even less new Factories